### PR TITLE
Rename master branch to main

### PR DESCRIPTION
**Pull request by vbauzysvmware**

### CODE REVIEW

**Review by lvirbalas — APPROVED** [review]

**Review by Didainius — COMMENTED** [review]
All is clear and thanks for that. I just used the opportunity to flag a few "stale" things which could probably be removed as well.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -30,4 +30,4 @@ an issue or PR.
 
 **Attribution**
 
-This Code of Conduct is adapted from the VMware Clarity project, available at this page: https://github.com/vmware/clarity/blob/master/CODE_OF_CONDUCT.md
+This Code of Conduct is adapted from the VMware Clarity project, available at this page: https://github.com/vmware/clarity/blob/main/CODE_OF_CONDUCT.md
```
[the remark this answers is not in the document]
Comment (lvirbalas) [reply CODE_OF_CONDUCT.md:33]: Is it here now?
https://github.com/vmware/clarity/blob/next/docs/CODE_OF_CONDUCT.md

[the remark this answers is not in the document]
Comment (vbauzysvmware) [reply CODE_OF_CONDUCT.md]: update to it

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# go-vcloud-director [![Build Status](https://travis-ci.org/vmware/go-vcloud-director.svg?branch=master)](https://travis-ci.org/vmware/go-vcloud-director) [![Coverage Status](https://coveralls.io/repos/vmware/go-vcloud-director/badge.svg?branch=master&service=github)](https://coveralls.io/github/vmware/go-vcloud-director?branch=master) [![GoDoc](https://godoc.org/github.com/vmware/go-vcloud-director?status.svg)](http://godoc.org/github.com/vmware/go-vcloud-director) [![Chat](https://img.shields.io/badge/chat-on%20slack-brightgreen.svg)](https://vmwarecode.slack.com/messages/CBBBXVB16)
+# go-vcloud-director [![Build Status](https://travis-ci.org/vmware/go-vcloud-director.svg?branch=main)](https://travis-ci.org/vmware/go-vcloud-director) [![Coverage Status](https://coveralls.io/repos/vmware/go-vcloud-director/badge.svg?branch=main&service=github)](https://coveralls.io/github/vmware/go-vcloud-director?branch=main) [![GoDoc](https://godoc.org/github.com/vmware/go-vcloud-director?status.svg)](http://godoc.org/github.com/vmware/go-vcloud-director) [![Chat](https://img.shields.io/badge/chat-on%20slack-brightgreen.svg)](https://vmwarecode.slack.com/messages/CBBBXVB16)
```
[the remark this answers is not in the document]
Comment (vbauzysvmware) [reply README.md]: removed

**Review by Didainius — APPROVED** [review]

**Pull request merged**